README.md

```diff
@@ -74,7 +74,10 @@ conda activate pyrado
 conda install -y blas cmake lapack libgcc-ng mkl mkl-include patchelf pip pycairo setuptools -c conda-forge
 pip install -r requirements.txt
 ```
-
+If you want to use the gym wrapper
+```
+pip install gym
+```
```

Comment:
I understand that you didn't add it to the requirements. Do you see any change we can test the wrapper using pytest?

Reply:
Well, there is the option of installing stable-baselines3 and use their [gym environment checker](https://stable-baselines3.readthedocs.io/en/master/common/env_checker.html). Other than that I think it is a bit difficult.

Reply:
I defintely dont want to install `stable-baselines` to test this wrapper. What I meant is to write pytests with a marker which checks if gym is importable. Just like for MuJoCo...

Reply:
Done in ac97a55